hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestHddsVolume.java

```diff
@@ -322,7 +322,7 @@ public void testReportUsedSmallerThanActualUsed() throws IOException {
     // actual used = 300(new allocates)
```

Comment:
Please update the javadoc of testReportUsedBiggerThanActualUsed() method too.

Reply:
Updated, PTAL.